### ISSUE TITLE
`output_bonds` not working

### ISSUE THREAD

**Issue by rkruegs123** [lorenzo-rovigatti/oxDNA#49]
I have recently upgraded to the latest commit and have tried to use the `output_bonds` analysis tool. However, for any topology file that I use (including those in the examples), I receive the message: `oxpy.core.OxDNAError: Can't read topology file 'initial.top'. Aborting`

I can successfully run `run.py` in `examples/OXPY` and pass the other build tests. An example run that gives me this error message is: `oat output_bonds examples/HAIRPIN/input examples/HAIRPIN/initial.conf`

**Comment by rkruegs123** [comment]
Another error that I receive is 
```
Traceback (most recent call last):
  File "/usr/lib/python3.9/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/home/ryan/.local/lib/python3.9/site-packages/oxDNA_analysis_tools/output_bonds.py", line 65, in compute
    print("{} {} {} {} {} {} {} {} {} {} {}".format(p, q, l[0], l[1], l[2], l[3], l[4], l[5], l[6], l[7], l[8]))
IndexError: index 8 is out of bounds for axis 0 with size 8
```

I think this comes from the assumption that there is a Debye-Huckel term. I am using oxDNA 1, so I have 7 subterms instead of 8

**Comment by lorenzo-rovigatti** [comment]
`oxpy` works exactly as `oxDNA`: it expects the paths found in the input file to either be absolute or relative to the current directory. I think that changing this behaviour would be very confusing (and somewhat arbitrary). What we can do is to add more options to alter the default behaviour. At the moment there is no way of setting the working directory from within `oxpy`, but I guess that Python's `os.chdir()` should do the trick, so that one option would be to add to `oat` a parameter that can be used to set the working directory with it.

However, before doing that we need to make sure that the solution

1. does not break other users' workflows
2. does not make the code to read the wrong files because of some ambiguous assumption

Any ideas?

**Comment by ErikPoppleton** [comment]
I assume there's nothing inside oxDNA which could easily set a working directory (defaulting to `.` if not set) to search for files relative to?  In that case, it wouldn't be too bad to add a keyword argument to all `oxpy`-using functions in `oat` which optionally sets the simulation directory to different than the directory where the Python interpreter is running.

**Comment by lorenzo-rovigatti** [comment]
There is `chdir` also in C (and C++). I could add it as an option to the input file, but that wouldn't help in this specific case. One option would be to add a `os.chdir(os.getcwd())` or something like this to `oxpy` or `oat`'s `__init__.py` file.

**Comment by ErikPoppleton** [comment]
I guess the question here is semantically where do we want the directory choice to be?  Is it
```python
output_bonds(traj_info, top_info, inputfile, directory="EXAMPLES/hairpin/")
```
and then run `os.chdir` in the Python function, or do we have something like
```python
inp = oxpy.InputFile()
inp.set_wd("EXAMPLES/hairpin/)
```
and then set it on the `oxpy`/oxDNA side.

**Comment by ErikPoppleton** [comment]
Fixed the oxDNA1 incompatibility with ee49f10d4c11ba4c61ef9f44558343dbe8781587

**Comment by rkruegs123** [comment]
Thank you so much for the prompt fix!
